tool/test.sh

```diff
@@ -1,14 +1,5 @@
-#!/bin/bash
-# Fast fail the script on failures.
+#!/bin/sh
 set -e
 
-DART_VERSION=$(dart --version 2>&1)
-DART_2_PREFIX="Dart VM version: 2"
-
-if [[ $DART_VERSION = $DART_2_PREFIX* ]]; then
-    echo -e 'pub run build_runner test -- -p chrome -p vm --reporter=expanded'
-    pub run build_runner test -- -p chrome -p vm --reporter=expanded
```

Comment:
We don't actually need build_runner here since there are no builders being applied... I think.